chemprop/uncertainty/predictor.py

```diff
@@ -1,12 +1,15 @@
 from abc import abstractmethod
 from typing import Iterable
+import copy
 
 from lightning import pytorch as pl
```

Comment:
```suggestion
import lightning as L
```

Reply:
For now I'll decide to not incorporate this change. If I use that style here, I should probably change it everywhere else in the code too. Open to discussing this more.

Reply:
fair enough--could be a separate PR but that's what the new lightning style recommends